scripts/create-kubestellar-demo-env.sh

```diff
@@ -134,14 +97,85 @@ checking_cluster() {
             
         else
             echo "$1 not found. Trying again..."
-            sleep 5
+            sleep 20
         fi
 
     done
 }
+##########################################
+
+echo -e "\nStarting environment clean up..."
+echo -e "Starting cluster clean up..."
+
+yq -i 'del(.preferences)' ${KUBECONFIG:-$HOME/.kube/config} 
+
+cluster_clean_up "kind delete cluster --name kubeflex" &
+cluster_clean_up "kind delete cluster --name cluster1" &
+cluster_clean_up "kind delete cluster --name cluster2" &
+wait
+echo -e "Cluster space clean up has been completed"
+
+echo -e "\nStarting context clean up..."
+context_clean_up
+echo "Context space clean up completed"
+
+echo -e "\nStarting the process to install KubeStellar core: kind-kubeflex..."
+clusters=(cluster1 cluster2)
+for cluster in "${clusters[@]}"; do
+   (
+     echo -e "Creating cluster ${cluster}..."
+     kind create cluster --name "${cluster}" >/dev/null 2>&1 &&
```

Comment:
The `>/devnull 2>&1`means that it is going to be hard for anyone to debug if anything goes wrong.

Reply:
yes, I initially went with this with normal output however, because the commands run in parallel, to save time, the output did not make sense anyway - it was just repeating each line (1 for each cluster). I think this is just a preference in the end, they are vanilla kind clusters. if this fails, we would be able to reproduce via cmd line

Reply:
The full complexity of the behavior of this script would be practically impossible to reproduce via command line.